pkg/cache/clusterqueue.go

```diff
@@ -680,21 +681,30 @@ func (c *ClusterQueue) UsedCohortQuota(fName kueue.ResourceFlavorReference, rNam
 	return cohortUsage
 }
 
-// DominantResourceShare returns a value from 0 to 100 representing the maximum of the ratios
+// DominantResourceShare returns a value from -100 to 100 representing the maximum of the ratios
```

Comment:
That would be for a CQ without any flavors. It's ok to ignore.

But it cannot generally go below -100, because the denominator is the total lendable resources in the cluster.